_posts/2024-01-19-trino-2023-wrapped.md

```diff
@@ -0,0 +1,253 @@
+cool new features and technologies you can use when you’re running Trino in
+
+<!--more-->
+
+## 2023 by the numbers
+
+* 64,288 views 👀 on YouTube
+* 5,872 hours watched ⌚on YouTube
+* 5,018 new commits 💻 in GitHub
+* 2,985 new stargazers ⭐ in GitHub
+* 2,494 pull requests merged ✅ in GitHub
+* 1,227 issues 📝 created in GitHub
+* 704 new subscribers 📺 in YouTube
+* 45 videos 🎥 uploaded to YouTube
+* 30 Trino 🚀 releases
+* 39 blog ✍️ posts
+* 10 Trino Community Broadcast ▶️ episodes
+* 2 Trino ⛰️ Summits
+
+We’re excited to say that Trino continued to grow in 2023 - GitHub stars
+increased by nearly 50% total and by 8% more than last year, commits increased
+by 7%, and Slack usage picked up dramatically. YouTube viewership was up 7%
+despite a lack of Pokemon-themed musical content compared to 2022 (our bad), and
+30 releases kept new versions of Trino coming out more than every other week.
+The Twitter/X account is no longer active for a few reasons that astute readers
+can hopefully infer, so it’s more important than ever to be
+on [our Slack](/slack.html). If you’re a Trino user or community member and
+aren’t already on there, you’re missing out! Make sure to join up for community
+announcements, release statuses, the shared expertise of the entire Trino
+community, and event-specific channels for discussion when we’re hosting things
+like Trino Fest and Trino Summit. Speaking of those…
+
+## Trino Events
+
+### Trino Fest and Trino Summit
+
+Trino Fest and Trino Summit are becoming mainstays on the Trino calendar each
+year, and 2023 was no different. Formerly “Cinco de Trino,” we ditched the Cinco
+de Mayo theme and went with the simpler “Trino Fest” in June, opting to theme it
+around Commander Bunbun’s Lake House Summer Camp, with a focus on integrating
+Trino with lakehouse and data lake architectures. Trino Summit only wrapped up a
+little over a month ago, rounding out the year and highlighting some of the
+amazing developments that we’ll be talking about later in this blog post.
+
+Trino Fest has historically been the smaller event, but it did some catching up
+in 2023, as both Trino Fest and Trino Summit were made virtual and expanded to 2
+days this year. Easier to attend than ever before, we reached a combined total
+of about 1,200 live attendees, with thousands more views on demand.
+
+The lineups were packed with 34 talks across both events, featuring speakers
+from huge Trino users like Salesforce, Stripe, Apple, and Lyft, as well as from
+major Trino contributors like Starburst, Tabular, and Bloomberg. You can
+view [recordings of every Trino Fest talk](https://www.youtube.com/playlist?list=PLFnr63che7wbBu_czq-SS9iVdQ4CIv2z1)
+and [every Trino Summit talk](https://www.youtube.com/playlist?list=PLFnr63che7wYeJLUjUaEftCFfjymhgLcq)
+on the Trino YouTube channel if you missed out.
+
+### Meetups and international events
+
+One of the more exciting developments was our first major event in Japan -
```

Comment:
wasnt a first at all .. just came back .. unless you dont count PrestoSQL events